integration_tests/integration/barcode/createBarcodeAddressImage.spec.ts

```diff
@@ -14,7 +14,7 @@ context('Create Barcode Image', () => {
     Page.verifyOnPage(ChooseBarcodeOptionPage).continueToImage()
     Page.verifyOnPage(GenerateBarcodeImagePage)
       .barcodeAddressImageExists()
-      .imageDownloadButtonExists(/Gage-Hewitt-A1234BC-\d{2}-\d{2}-\d{4}\.png/)
+      .imageDownloadButtonExists(/SendLegalMail-Gage-Hewitt-\d{4}-\d{2}-\d{2}\.png/)
```

Comment:
I don't know the context of Gage-Hewitt as a test case, but it feels like a specific person. Could we consider using a more generic, or less specific name?

Reply:
Thanks @joelstobart-moj - I'll bring it up with the rest of the team, but the design team have been using Gage Hewitt and John Smith as their 2 recipients in all of their screen designs and mock ups etc since the beginning of the project. I'd always assumed there were fictitious people, but I guess any name could end up being the name of a real person at some stage? 🤷‍♂️

Reply:
I think the problem with the Gage one is that it's specific enough to be personally identifying, or at least wrongly personally identifying. E.g. http://howmanyofme.com/search/ says "There are 1 or fewer people in the U.S. named Gage Hewitt.". 

John Smith / John Doe / Jane Doe / [Ashok Kumar](https://www.cnbctv18.com/legal/what-is-john-doe-or-ashok-kumar-order-390051.htm) are all recognised placeholders.